src/bundle/DependencyInjection/Compiler/FieldTypeProcessorPass.php

```diff
@@ -6,12 +6,17 @@
  */
 namespace EzSystems\EzPlatformRestBundle\DependencyInjection\Compiler;
 
+use eZ\Publish\Core\Base\Container\Compiler\TaggedServiceIdsIterator\BackwardCompatibleIterator;
+use LogicException;
 use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\DependencyInjection\Reference;
 
 class FieldTypeProcessorPass implements CompilerPassInterface
 {
+    public const FIELD_TYPE_PROCESSOR_SERVICE_TAG = 'ibexa.rest.field_type.processor';
+    public const DEPRECATED_FIELD_TYPE_PROCESSOR_SERVICE_TAG = 'ezpublish_rest.field_type_processor';
```

Comment:
Since it's deprecated and nobody would have anywhere at this point, won't it make sense to make it private immediately?

```suggestion
    private const DEPRECATED_FIELD_TYPE_PROCESSOR_SERVICE_TAG = 'ezpublish_rest.field_type_processor';
```

We could use the constant above as `!php/const` in YAML files, but that's kinda nitpicky I guess?

Reply:
The main purpose of this constants is to avoid hardcoding tag names in *.php (including tests), and only there :wink:

Reply:
@adamwojs I'd say that the purpose of the tests is to detect a change in constant values in this case. Similarly how hardcoding URLs for application tests is recommended.

If we're concerned that changing this tag will force us to change values in multiple files, then using `!php/const` makes more sense than in tests actually, as it's occuring there a lot more than in tests.

Reply:
Addresses in https://github.com/ezsystems/ezplatform-rest/pull/64